float64[3] fk_config_tran
float64[3] fk_config_rot
float64[3] ik_target_coord
---
float64[3] tip_coord
float64[3] config_tran
float64[3] config_rot